pypeit/scripts/chk_edges.py

```diff
@@ -10,65 +10,25 @@
 import argparse
 
 def parser(options=None):
-
-    parser = argparse.ArgumentParser(description='Display MasterTrace image in a previously launched RC Ginga viewer',
+    parser = argparse.ArgumentParser(description='Display MasterEdges image and trace data',
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument('root', type=str, default = None, help='PYPIT Master Trace file root [e.g. MasterTrace_A_01_aa.fits]')
-    parser.add_argument("--chname", default='MTrace', type=str, help="Channel name for image in Ginga")
-    parser.add_argument("--dumb_ids", default=False, action="store_true", help="Slit ID just by order?")
-    parser.add_argument("--old", default=False, action="store_true", help="Used old slit tracing")
+    parser.add_argument('trace_file', type=str, default = None,
+                        help='PypeIt Master Trace file [e.g. MasterEdges_A_01_aa.fits.gz]')
+    parser.add_argument('--chname', default='MTrace', type=str,
+                        help='Channel name for image in Ginga')
     parser.add_argument('--mpl', default=False, action='store_true',
                         help='Use a matplotlib window instead of ginga to show the trace')
 
     return parser.parse_args() if options is None else parser.parse_args(options)
 
 
 def main(pargs):
```

Comment:
Nice this looks a hell of a lot cleaner!

Reply:
Mostly because I pushed the connecting to ginga stuff into `edgetrace`.